### PR TITLE
cleanup cpu base building script

### PR DESCRIPTION
**Pull request by aslonnie**
get prepare to replace the building steps with wanda`

### CODE REVIEW

**Comment by aslonnie** [comment]
proof of test: https://buildkite.com/ray-project/oss-ci-base-image-builder/builds/384#0189de4c-c21a-4ec9-86d9-a19b457a9ccd

File: ray_ci/build_base_build.sh [hunk 1]
```diff
@@ -1,89 +1,98 @@
...
-export DOCKER_BUILDKIT=1
+set -euo pipefail
 
-cd "$RAY_REPO_DIR" || true
+export DOCKER_BUILDKIT=1
 
-if [ "$BUILDKITE_COMMIT" = "HEAD" ]; then
-  export BUILDKITE_COMMIT=$(git log -1 --format="%H")
-  echo "Resolved BUILDKITE_COMMIT to $BUILDKITE_COMMIT"
+if [[ "${RAY_REPO_DIR:-}" != "" ]]; then
```
Comment (can-anyscale) [inline ray_ci/build_base_build.sh:7]: is this a new env introduced in this PR? does it matter?

**Review by can-anyscale — APPROVED** [review]

File: ray_ci/build_base_build.sh (hunk 1 above)
Comment (aslonnie) [reply ray_ci/build_base_build.sh:7]: it was referenced on line 6? I just trying to make it run with `set -u` even when it is running locally.

this var is set to specify where the ray repo is checked out.

this script is a legacy script that is being used today on civ1. I am doing the refactoring to make it easier to co-exist with civ2.

**Pull request merged**